src/test/scala-2.11/rx/lang/scala/completeness/ObservableCompletenessKit.scala

```diff
@@ -76,6 +76,8 @@ class ObservableCompletenessKit extends CompletenessKit {
     "doOnUnsubscribe(Action0)" -> "doOnUnsubscribe(=> Unit)",
     "doOnTerminate(Action0)" -> "doOnTerminate(=> Unit)",
     "elementAtOrDefault(Int, T)" -> "elementAtOrDefault(Int, U)",
+    "extend(Func1[_ >: OnSubscribe[T], _ <: R])" -> "[TODO]",
+    "fromCallable(Callable[_ <: T])" -> "[TODO]",
```

Comment:
Can't we just use `Observable.defer(Observable.just(expensiveComputation))`? If yes, can you add an example for this to `RxScalaDemo`?